client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -332,16 +332,16 @@ class DomainManagementNavigationEnhanced extends React.Component {
 	}
 
 	getSimilarDomains() {
-		const { selectedSite, translate } = this.props;
+		const { domain, selectedSite, translate } = this.props;
+		const searchTerm = domain.name.split( '.' )[ 0 ];
```

Comment:
Do you mind putting a comment here on why we're doing it like that (because of the warning in the search)